docs/en/bridges.wml

```diff
@@ -228,30 +265,52 @@
     <br>
     <img src="$(IMGROOT)/tb-tor-button-menu.png" alt="Tor Browser's TorButton Menu">
     <br><br><br>
-    <h3>3) Select "My Internet Service Provider (ISP) blocks connections to the Tor network":</h3>
+    <h3>3) Select "My Internet Service Provider (ISP) blocks connections to the 
+        Tor network":</h3>
     <br>
-    <img src="$(IMGROOT)/tb-bridge-networksettings.png" alt="Tor Browser's Network Settings page">
+    <img src="$(IMGROOT)/bridges/tb-tor-launcher-bridges.png"
+      alt="Tor Browser's Network settings" />
     <br><br><br>
-    <h3>4) Now you have two configuration options. You can use bridges which are
-        preconfigured and provided with Tor Browser, or you can specify your
-        own bridge(s).</h3>
+    <h3>4) Now you have three configuration options. You can use bridges which
+        are preconfigured and provided with Tor Browser, you can request new
+        bridges, or specify your own bridge(s).</h3>
     <br>
-    <img src="$(IMGROOT)/tb-bridges-options-from-browser.png" alt="Tor Browser's TorButton Menu">
+    <img src="$(IMGROOT)/bridges/tb-tor-launcher-bridges.png"
+      alt="Tor Browser's Bridge Configuration page">
     <br><br><br>
-    <h3>5a) If you want to use one of the provided bridges, then choose the
-    transport type you want to use. obfs4 is currently the recommend
-    type, but depending on where you are located another type may work better
-    for you. If you have any questions, please <a href="<page about/contact>#support">contact
-    us.</a></h3>
-    <br>
-    <img src="$(IMGROOT)/tb-bridges-provided-from-browser.png" alt="My Internet Service Provider (ISP) blocks connections to the Tor network">
+    <h3>5a) If you want to use one of the provided bridges, then choose one of
+        the offered transport types. obfs4 is currently recommend, but
+        depending on where you are located another one may work better for you.
+        If you have any questions, please
+        <a href="<page about/contact>#support">contact us</a>.</h3>
```

Comment:
Change from contact page to support.torproject.org

Reply:
done.